plugin/plugin.go

```diff
@@ -78,8 +78,12 @@ const (
 	EventPlugin Type = "io.containerd.event.v1"
 	// LeasePlugin implements lease manager
 	LeasePlugin Type = "io.containerd.lease.v1"
+	// Streaming implements a stream manager
+	StreamingPlugin Type = "io.containerd.streaming.v1"
```

Comment:
Will this cause confusion with stream processors?
I don't come up with a better name though 😅

Reply:
I think if you are deep enough into containerd's architecture to be using stream processors it won't be too hard to understand this difference and find where it is documented. Confusion is hard to avoid but we can always attempt to address it in documentation if it becomes a frequently asked question. Also interesting here is you could potentially use the streaming API to do client side stream processors, so many there is some relationship in the future.